modin/core/execution/unidist/common/utils.py

```diff
@@ -34,6 +32,7 @@ def initialize_unidist():
         )
 
     if not unidist.is_initialized():
+        unidist_cfg.CpuCount.put(modin_cfg.CpuCount.get())
```

Comment:
```suggestion
        modin_cfg.CpuCount.subscribe(lambda count: unidist_cfg.CpuCount.put(count.get()))
```
now that I think of it, this should be all that is needed to configure unidist and keep it in sync with `modin_cfg` 😉